taskcluster/examples-mic_vad_streaming-py38.yml.DISABLED_UNTIL_SCIPY_PY38

```diff
@@ -0,0 +1,13 @@
+build:
```

Comment:
Why is this disabled?

Reply:
Because there is not yet a scipy for 3.8